visualization/index.js

```diff
@@ -13,7 +13,7 @@ const path= require('path')
 app.use(express.static('dist'))
 app.use(express.json())
 app.use(function (req, res, next) {
-  res.setHeader('Access-Control-Allow-Origin', 'http://localhost:3000');
+  res.setHeader('Access-Control-Allow-Origin', 'http://localhost:3001');
```

Comment:
See docker compose comment above: potentially a good place for an environment variable

Reply:
Good idea, new issue #72